internal/pkg/deploy/cloudformation/app.go

```diff
@@ -312,6 +312,51 @@ func (cf CloudFormation) RemoveJobFromApp(app *config.Application, jobName strin
 	return nil
 }
 
+// RemoveEnvFromAppOpts contains the parameters to call RemoveEnvFromApp.
+type RemoveEnvFromAppOpts struct {
+	App                 *config.Application
+	EnvName             string
+	EnvAccountID        string
+	EnvRegion           string
+	DeleteStackInstance bool // If this is the last environment in a region, we can delete it.
+}
+
+func (cf CloudFormation) RemoveEnvFromApp(opts *RemoveEnvFromAppOpts) error {
```

Comment:
Comment pls 🙏 If i understand correctly the only thing we need to do for `RemoveEnvFromApp` should be optionally delete the stackset instance? Also sorry this might be a stupid question but I'm not sure how we figure out this is the last environment we try to delete from this account/region...